lib/src/realm/scene_entity_coordinator.rs

```diff
@@ -77,6 +113,37 @@ impl SceneEntityCoordinator {
         self.dirty_loadable_scenes = true;
     }
 
+    fn do_request(&mut self, request_option: RequestOption) {
+        if self.http_requester.is_none() {
+            self.http_requester = Some(Arc::new(HttpQueueRequester::new(10, None)));
+        }
+
+        let sender = self.sender.clone();
+        let http_requester = self.http_requester.clone().unwrap();
+
+        if let Some(rt) = self.runtime.as_ref() {
+            rt.spawn(async move {
+                let result = http_requester.request(request_option, 0).await;
+                if let Err(error) = sender.send(result).await {
+                    tracing::error!("Error sending the result: {:?}", error);
+                }
+            });
+        } else {
+            std::thread::spawn(move || {
+                let runtime = tokio::runtime::Runtime::new();
+                if runtime.is_err() {
+                    panic!("Failed to create runtime {:?}", runtime.err());
+                }
+                let runtime = runtime.unwrap();
+
+                runtime.block_on(async move {
+                    let result = http_requester.request(request_option, 0).await;
+                    let _ = sender.try_send(result);
+                });
+            });
```

Comment:
Do you think it’s worth maintaining this fallback? 🤔 Maybe I should just log an error instead, as the fallback code is very slow if used frequently.

Reply:
only for testing